webpack.config.babel.js

```diff
@@ -233,6 +233,9 @@ const baseConfig = {
       '~locales': `${__dirname}/src/locales`,
       '~types': `${__dirname}/src/types`,
       '~utils': `${__dirname}/src/components/utils`,
+      'socket.io-client': resolve(
+        'node_modules/socket.io-client/dist/socket.io.js'
```

Comment:
On IE11 the SDK was failing to render with a syntax error in the console after upgrading to v4 due to the ES6 module version of `socket.io-client` being loaded instead of the compiled version. Basically same problem as [this issue](https://github.com/socketio/socket.io-client/issues/1457) that was raised in their project.